MoyaResponse.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  File.swift
```

Comment:
Actually, you know what? Just leave them. They're all in the other files, too, so I'd rather make them consistent in the same commit.